config/locales/client.en.yml

```diff
@@ -2546,6 +2546,10 @@ en:
     not_logged_in_user: "user page with summary of current activity and preferences"
     current_user: "go to your user page"
     view_all: "view all %{tab}"
+    user_menu:
+      generic_no_items: "There are no items in this list."
+      sr_menu_tabs: "Menu tabs"
```

Comment:
What does `sr` stand for here? I think we might want to consider just spelling out the word to avoid any ambiguity.

Reply:
`sr` stands for "screen reader". I believe we have a convention/pattern of prefixing translations that are used for screen readers with `sr_`? There are certainly a number of existing strings that follow this pattern:

```
➜  grep 'sr_' config/locales/client.en.yml
      sr_expand_profile: "Expand profile details"
      sr_collapse_profile: "Collapse profile details"
        sr_title: "Sign in with Google"
        sr_title: "Sign in with Twitter"
        sr_title: "Log in with Instagram"
        sr_title: "Log in with Facebook"
        sr_title: "Log in with GitHub"
        sr_title: "Log in with Discord"
      sr_menu_tabs: "Menu tabs"
      sr_collapse_replies: "Collapse embedded replies"
      sr_date: "Post date"
      sr_expand_replies:
      sr_below_embedded_posts_description: "post #%{post_number} replies"
      sr_embedded_reply_description: "reply by @%{username} to post #%{post_number}"
      sr_post_like_count_button:
      sr_post_read_count_button:
          sr_post_likers_list_description: "users who liked this post"
          sr_post_readers_list_description: "users who read this post"
    sr_views: "Sort by views"
    sr_replies: "Sort by replies"
    sr_activity: "Sort by activity"
    sr_likes: "Sort by likes"
    sr_op_likes: "Sort by original post likes"
      sr_jump_top_button: "Jump to the first post"
      sr_jump_bottom_button: "Jump to the last post"
```

Reply:
I don't really like the convention but I guess the ship has sailed on this one :ship: Let us leave this as it is then, thank you for clarifying :+1: